Support/include/ofxsProcessing.h

```diff
@@ -3,37 +3,9 @@
 
 /*
   OFX Support Library, a library that skins the OFX plug-in API with C++ classes.
-  Copyright (C) 2005 The Open Effects Association Ltd
+  Copyright OpenFX and contributors to the OpenFX project.
```

Comment:
This can be simplified to....

'Copyright contributors to the OpenFX project'

Reply:
The ASWF guidance is what I referenced... https://tac.aswf.io/process/contributing.html#copyright-notice-format